release notes/v0.47.0.md

```diff
@@ -37,6 +37,16 @@ Thanks, @sapphire-janrain, for contribution!
 
 In some workflows the reflection call should also include some metadata. This PR adds [a new connection parameter `reflectMetadata`](https://k6.io/docs/javascript-api/k6-net-grpc/client/client-connect/#connectparams) that allows to specify the metadata to be sent with the reflection call.
 
+### Improved the browser module's cookie API
```

Comment:
Should the list of PRs be added to the end of this? Then we can shorten this to something like

```suggestion
Improve the cookie API
```

This suggestion is going by what i see elsewhere in the release notes.

Reply:
As far as I can see, a big new feature is written as a heading; like we did with Web Vitals [before](https://github.com/grafana/k6/blob/39f0e6882228a061dbc495344dda5cc1261e7942/release%20notes/v0.44.0.md). Or, do you mean something else? 🤔

Reply:
I'm not 100% sure on shortening the heading tbh.

What about adding the list of issues after the heading, which seems to be how other new features are added.

```suggestion
### Improved the browser module's cookie API browser#1, browser#2...
```

Reply:
Fair enough. Last suggestion on this 🙂 What about adding the issue links closer to when they're first mentioned e.g.:

```suggestion
The browser module now provides a more complete and robust API for handling cookies. We stabilized the cookie API by defining a new [`Cookie` class](https://k6.io/docs/javascript-api/k6-experimental/browser/browsercontext/cookie) (browser#1, browser#2)
```

and

```suggestion
We added a new [`browser.cookies([urls])`](https://k6.io/docs/javascript-api/k6-experimental/browser/browsercontext/cookies/) (browser#3) method that returns all cookies from the current [browser context](https://k6.io/docs/javascript-api/k6-experimental/browser/browsercontext). The new API also supports filtering cookies by URL (browser#4).
```

Then we wouldn't need the related link block at the end of this section, which doesn't feel like it fits with how we generally incorporate the pr/issue links.

Reply:
Can you write it as a suggestion because I'm not sure I understand 😮‍💨

Reply:
See these:

- https://github.com/grafana/k6/pull/3352/files#r1337163681
- https://github.com/grafana/k6/pull/3352/files#r1337163639

Reply:
Thanks! Check it out: [38c57bc](https://github.com/grafana/k6/pull/3352/commits/38c57bce967386f000b8b937496fb3985d47f470)